options/posix/generic/sched-stubs.cpp

```diff
@@ -31,18 +32,31 @@ int sched_get_priority_max(int) {
 }
 
 int sched_get_priority_min(int) {
-	__ensure(!"Not implemented");
-	__builtin_unreachable();
+	mlibc::infoLogger() << "mlibc: sched_get_priority_min() returns ENOSYS" << frg::endlog;
+	errno = ENOSYS;
+	return -1;
 }
 
 int __mlibc_cpu_isset(int, cpu_set_t *) {
 	__ensure(!"Not implemented");
 	__builtin_unreachable();
 }
 
-int __mlibc_cpu_count(cpu_set_t *) {
-	__ensure(!"Not implemented");
-	__builtin_unreachable();
+int __mlibc_cpu_count(const cpu_set_t *set) {
+	size_t i = 0;
+	size_t bit = 0;
+	size_t count = 0;
+
+	const unsigned char *ptr = reinterpret_cast<const unsigned char *>(set);
+
+	for(i = 0; i < sizeof(cpu_set_t); i++) {
+		for(bit = 0; bit < CHAR_BIT; bit++) {
+			if((1 << bit) & ptr[i])
+				count++;
+		}
+	}
+
+	return count;
```

Comment:
this doesn't match our style guidelines (and is also a very weird way to implement a popcnt). why are the loop counters declared out of line and initialized twice?